### DISCUSSION TITLE
Landing the a320x

### DISCUSSION THREAD

**Discussion by hausnm** [flybywiresim/aircraft#3671] [Help] [answered]
Hello,
Today I tested the flybywire mod of the 320. during ILS landing suddenly the warning too low terrain called out. I had gears down and flaps full. ILS was on, Autopilot was turned of, throttle almost idle. I have been just right near to the runway. Is it a bug or what else do I have to do ? Thanks for your help.

**Comment by hotshotp** [comment, marked as answer]
being worked on
